components/tabstrip/overview.md

```diff
@@ -55,28 +41,38 @@ To use a Telerik TabStrip for Blazor:
 }
 ````
 
->caption The result from the code snippet above
+## Active Tab Index
 
-![Blazor Tabstrip Left](images/tabstrip-left.png)
+By default, the initially selected tab is the first one. You can programmatically control the selected tab through the `ActiveTabIndex` parameter. [Read more about the `ActiveTabIndex`...]({%slug tabstrip-activetabindex%})
 
+## Tabs Configuration
```

Comment:
You can add something more specific to increase the value of the section and paragraph (e.g. mention the three tab parameters, since they are a small number anyway)